core-it-suite/src/test/java/org/apache/maven/it/MavenITmng3092VersionRangeResultFilterExtensionTest.java

```diff
@@ -0,0 +1,132 @@
+package org.apache.maven.it;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.apache.maven.it.Verifier;
+import org.apache.maven.it.util.ResourceExtractor;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This is a test set for <a href="https://issues.apache.org/jira/browse/MNG-3092">MNG-3092</a>.
+ *
+ * <pre>
+ *   <dependencies>
+ *     <dependency>
+ *       <groupId>org.apache.maven.its.mng3092</groupId>
+ *       <artifactId>a</artifactId>
+ *       <version>[1.0,2.0)</version>
+ *     </dependency>
+ *   </dependencies>
+ * </pre>
+ *
+ * @author Benjamin Bentmann
+ */
+public class MavenITmng3092VersionRangeResultFilterExtensionTest
+        extends AbstractMavenIntegrationTestCase
+{
+
+    public MavenITmng3092VersionRangeResultFilterExtensionTest()
+    {
+        super( "[3.4.0-SNAPSHOT,)" );
```

Comment:
Am 2016-06-21 um 22:28 schrieb barthel:
>> + *       <groupId>org.apache.maven.its.mng3092</groupId>
>> + *       <artifactId>a</artifactId>
>> + *       <version>[1.0,2.0)</version>
>> + *     </dependency>
>> + *   </dependencies>
>> + * </pre>
>> + *
>> + * @author Benjamin Bentmann
>> + */
>> +public class MavenITmng3092VersionRangeResultFilterExtensionTest
>> +        extends AbstractMavenIntegrationTestCase
>> +{
>> +
>> +    public MavenITmng3092VersionRangeResultFilterExtensionTest()
>> +    {
>> +        super( "[3.4.0-SNAPSHOT,)" );
>
> So this test will not execute before 3.4.0 release. I thought I change that close after the 3.4.0 release?
> If this is not a problem, I'll change this.

It will, see:
mng3099SettingsProfilesWithNoPom(it).................OK (1.2 s)
mng3092VersionRangeResultFilterExtension(Default)....OK (1.4 s)

Try it yourself. The -SNAPSHOT is ignored here because we mostly test 
snapshots.